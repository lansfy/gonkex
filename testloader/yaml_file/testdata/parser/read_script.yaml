- name: loader MUST read tests with script data
  method: GET
  path: /some/path
  response:
    200: "aaabbb"
  beforeScript:
    path: script1.sh
    timeout: 10
  afterRequestScript:
    path: script2.sh
    timeout: 500ms
