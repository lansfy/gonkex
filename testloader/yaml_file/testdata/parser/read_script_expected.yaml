- GetName: loader MUST read tests with script data
  GetMethod: GET
  Path: /some/path
  GetResponses:
    200: "aaabbb"
  BeforeScript:
    CmdLine: script1.sh
    Timeout: 10s
  AfterRequestScript:
    CmdLine: script2.sh
    Timeout: 500ms
  GetFileName: testdata/parser/read_script.yaml
  FirstTestInFile: true
  LastTestInFile: true
